src/Algebra/Bundles.agda

```diff
@@ -759,16 +790,17 @@ record KleeneAlgebra c ℓ : Set (suc (c ⊔ ℓ)) where
     _≈_                   : Rel Carrier ℓ
     _+_                   : Op₂ Carrier
     _*_                   : Op₂ Carrier
+    _⁻*                   : Op₁ Carrier
```

Comment:
I assume that `⁻*` is being used because `*` is already taken? I wonder if using `⋆` (i.e. \star) would be a better alternative? Mirrors the pronunciation and easier to type?